src/briefcase/platforms/linux/system.py

```diff
@@ -1038,21 +1038,25 @@ def _package_pkg(self, app: AppConfig, **kwargs):
 with details about the release.
 """
             )
-
+            
         # Generate the pkgbuild layout
-        pkgbuild_path = self.bundle_path(app) / "pkgbuild"
+        pkgbuild_path = self.build_path(app) / "pkgbuild"
         with self.input.wait_bar("Generating pkgbuild layout..."):
             if pkgbuild_path.exists():
                 self.tools.shutil.rmtree(pkgbuild_path)
             (pkgbuild_path).mkdir(parents=True)
 
             # Build the source archive
             with self.input.wait_bar("Building source archive..."):
+                
+                # Copy the CHANGELOG file to the bundle_path so it can be included in the source archive
+                self.tools.shutil.copy(changelog_source, self.bundle_path(app) / "CHANGELOG")
```

Comment:
Why does it need to be copied? Couldn't you add a single file directly to the archive?

Reply:
Since we are archiving the whole `bundle_path` directory, I thought first copying and then archiving would be better.

Reply:
Copying is a last resort; by copying, you're creating a duplicate file, which could lead to drift between the source and the packaged artefact. Packaging the source file directly is vastly preferred, and I can't see any reason it couldn't be done here.

Reply:
Sure. I will change it.